pkg/backend/filestate/backend.go

```diff
@@ -254,7 +254,7 @@ func (b *localBackend) SupportsOrganizations() bool {
 }
 
 func (b *localBackend) ParseStackReference(stackRefName string) (backend.StackReference, error) {
-	return localBackendReference{name: tokens.QName(stackRefName)}, nil
+	return localBackendReference{name: tokens.AsName(stackRefName)}, nil
```

Comment:
Little concerning that we've always just cast the string to QName here. While client code hopefully called ValidateStackName before calling this nothing required that. Thinking it wouldn't hurt to add here and would save an assert panic if there are any mistakes.